_posts/2018-05-16-strimzi-0-4-0-release.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Strimzi 0.4.0 released!"
+date: 2018-04-21
+---
+
+We are really happy to announce the new Strimzi 0.4.0 release with a lot of fixes and new interesting features!
+
+<!--more-->
+
+# Enhanced configuration
+
+Until the previous release, just few parameters were exposed through the cluster config map so that the Kafka brokers and Zookeeper nodes configurations were limited. With this release, the `kafka-config` ([doc](http://strimzi.io/docs/0.4.0/#kafka_configuration_json_config)) and the `zookeeper-config` ([doc](http://strimzi.io/docs/0.4.0/#zookeeper_configuration_json_config)) fields were added in order to allow the user to define a full configuration through a JSON string.
```

Comment:
We should mentioned Connect here as well.

Reply:
right!